spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/TaskAppNode.java

```diff
@@ -16,22 +16,34 @@
 
 package org.springframework.cloud.dataflow.core.dsl;
 
+import java.util.Collections;
+import java.util.LinkedHashMap;
 import java.util.List;
+import java.util.Map;
 
 /**
- * Common AST base class for nodes representing job definitions or job references.
+ * Represents either a task application or task definition reference. Task application references
```

Comment:
is this still applicable for "task definition" references, or only "task application"?